src/replica/replica_stub.cpp

```diff
@@ -789,6 +775,37 @@ void replica_stub::initialize(const replication_options &opts, bool clear /* = f
     }
 }
 
+void replica_stub::initialize_fs_manager(std::vector<std::string> &data_dirs,
+                                         std::vector<std::string> &data_dir_tags)
+{
+    std::string cdir;
+    std::string err_msg;
+    int count = 0;
+    std::vector<std::string> available_dirs;
+    std::vector<std::string> available_dir_tags;
```

Comment:
maybe you should assert data_dirs.size() == data_dir_tags.size() here

Reply:
data_dirs and data_dirs_tags are `_options.data_dirs` and `_options.data_dir_tags`, you can reference how them set in `replication_common.cpp` function `initialize`, I don't add a link here because it will be refactored by pull request 831.